bark/world/world.cpp

```diff
@@ -27,7 +27,10 @@ World::World(const commons::ParamsPtr& params)
       remove_agents_(params->GetBool(
           "World::remove_agents_out_of_map",
           "Whether agents should be removed outside the bounding box.",
-          false)) {
+          false)),
+      frac_lateral_offset_(params->GetReal("World::FracLateralOffset",
```

Comment:
Gäbe es eine Möglichkeit, dass je nach BehaviorModel unterschiedlich parametrierbar zu machen?

Dadurch könnte man "nettere" und "weniger" nette Fahrer einfach über den Parameter definieren.

Ansonsten passt es aber für mich so.